.github/workflows/cd-pipeline.yml

```
@@ -142,7 +142,10 @@ jobs:
             git reset --hard $(git commit-tree origin/index^{tree} -m "Update index.yaml")
 
             # Rebuild index
-            helm repo index --url https://charts.bitnami.com/bitnami --merge bitnami/index.yaml ../download
+            helm repo index --url oci://registry-1.docker.io/bitnamicharts/ --merge bitnami/index.yaml ../download
```

Is the final forward slash required?
```suggestion
            helm repo index --url oci://registry-1.docker.io/bitnamicharts --merge bitnami/index.yaml ../download
```
Just asking